app/Services/Phoenix/Phoenix.php

```diff
@@ -48,4 +48,22 @@ public function getCampaign($id)
 
         return is_null($response) ? null : $response;
     }
+
+    /**
+     * Send a GET request to return a reportback & an item with the specified id.
+     *
+     * @param  string $reportback_id
+     * @param  string $reportback_item_id
+     * @return object|null
+     */
+    public function getSpecificReportbackItem($reportback_id, $reportback_item_id)
```

Comment:
the pattern we've been using based on original Northstar pattern is to just use `getReportbackItem($id)` to imply retrieving a single resource and `getAllReportbackItems($params = [])` would be for a collections of resources.